### PR TITLE
Collapsible: Fixing icons for 1.5

### PR DESCRIPTION
**Pull request by arschmitz**

### CODE REVIEW

File: demos/collapsible/index.php [hunk 1]
```diff
@@ -225,8 +225,10 @@
 				<div data-demo-html="true">
 					<div data-role="collapsible" data-enhanced="true" class="ui-collapsible ui-collapsible-inset ui-corner-all ui-collapsible-collapsed">
 						<h4 class="ui-collapsible-heading ui-collapsible-heading-collapsed">
-							<a href="#" class="ui-collapsible-heading-toggle ui-button ui-icon-plus">
-							Heading
+							<a href="#" class="ui-collapsible-heading-toggle ui-button">
+                            <span class="ui-icon ui-icon-plus"></span>
+
```
Comment (gabrielschulhof) [inline demos/collapsible/index.php]: This blank line seems unnecessary, and there are too few tabs indenting the span in the previous line.

File: js/widgets/collapsible.js [hunk 2]
```diff
@@ -174,12 +174,19 @@ $.widget( "mobile.collapsible", {
 			.find( "a" )
 				.first()
 					.addClass( "ui-button " +
-						( iconclass ? iconclass + " " : "" ) +
-						( iconclass ? iconposClass( opts.iconpos ) +
-						" " : "" ) +
 						this._themeClassFromOption( "ui-button-", opts.theme ) + " " +
 						( opts.mini ? "ui-mini " : "" ) );
 
+		ui.icon = $( "<span class='ui-icon " + ( iconclass ? iconclass + " " : "" ) + "'></span>" );
```
Comment (gabrielschulhof) [inline js/widgets/collapsible.js]: You need to assign ```ui.icon``` by retrieving the span via a selector in the enhanced case.

Comment (gabrielschulhof) [inline js/widgets/collapsible.js]: https://github.com/arschmitz/jquery-mobile/blob/1.5-collapsible-icon/js/widgets/collapsible.js#L85

**Comment by arschmitz** [comment]
@gabrielschulhof can you take another look at this fixed a bunch of things i noticed

**Comment by arschmitz** [comment]
closing this is not based on the correct branch

**Pull request closed without merging**